README.md

```
@@ -1,337 +1,288 @@
-
-## Palavras-chave
-
-Se você quer que uma variável nunca seja alterada deve-se usar final ou const, em vez de usar var ou somente o tipo. Uma variável definida como final pode ter seu valor definido apenas uma vez, já uma variável const é uma constante de tempo de compilação. (Uma variável const é implicitamente final)
-
-Se a constante estiver em uma classe deve ser marcado como static const, assim a mesma variável será compartilhada por todas as instâncias da classe e também ser acessível sem instanciar a classe.
-
-    final name = 'Bob';
-    final String nickname = 'Bobby';
+# 🚀 Rick and Morty App
 
+<div align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)